ubcpi/static/html/ubcpi.html

```diff
@@ -158,6 +160,8 @@ <h2 id="pi-question-h" class="question-text">Question</h2>
                 <div class="detailed-solution" id="detailed-solution">
 
                     <h2 id="correct-option-head" ng-if="options.length > rc.correct_answer">Correct Answer</h2>
+                    <h2 id="correct-option-head" ng-if="!rc.correct_rationale.text"></h2>
```

Comment:
That is for not displaying any heading to the student in the case where the instructor has chosen to not specify a correct answer ("n/a"). For that case only, we make the rationale field optional, so that the instructor can also choose to not specify a "correct" rationale.